### PR TITLE
IPOR Protocol - Arbitrum swaps volume

### PR DESCRIPTION
**Pull request by amydlarz**

### CODE REVIEW

**Comment by llamatester** [comment]
The ipor adapter exports: 
        
 
 _______________________________________
    Dexs for 26/2/2024
    _______________________________________
    
    Version -> DERIVATIVES
    ---------
    ETHEREUM 👇
    Backfill start time: 16/8/2022
    NO METHODOLOGY SPECIFIED
    Timestamp: 1708991999 (2024-02-26T23:59:59.000Z)
    Daily volume: 0
    
    
    ARBITRUM 👇
    Backfill start time: 21/2/2024
    NO METHODOLOGY SPECIFIED
    Timestamp: 1708991999 (2024-02-26T23:59:59.000Z)
    Daily volume: 0
    
    
    Done in 8.13s.

**Comment by amydlarz** [comment]
For 2024-02-26 returns balance
```
{
  chain: 'arbitrum',
  timestamp: undefined,
  _balances: { '0x5979D7b546E38E414F7E9822514be443A4800529': 3.9447389885807502 }
}
```
but daily volume is 0. Is it correct?

File: dexs/ipor/index.ts [hunk 1]
```diff
@@ -38,7 +44,7 @@ const fetch: any = async (timestamp: number, _: any, { chain, getLogs, createBal
 
   logs.forEach(log => {
     const balance = Number(log.money?.notional || log.amounts?.notional)
-    dailyNotionalVolume.add('tether', balance / 1e18, { skipChain: true })
+    dailyNotionalVolume.addToken(log.asset, balance / 1e18, { skipChain: true })
```
Comment (dtmkeng) [inline dexs/ipor/index.ts]: fix this with `dailyNotionalVolume.add(log.asset, balance)`

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The ipor adapter exports: 
        
 
 _______________________________________
    Dexs for 27/2/2024
    _______________________________________
    
    Version -> DERIVATIVES
    ---------
    ETHEREUM 👇
    Backfill start time: 16/8/2022
    NO METHODOLOGY SPECIFIED
    Timestamp: 1709078399 (2024-02-27T23:59:59.000Z)
    Daily volume: 0
    
    
    ARBITRUM 👇
    Backfill start time: 21/2/2024
    NO METHODOLOGY SPECIFIED
    Timestamp: 1709078399 (2024-02-27T23:59:59.000Z)
    Daily volume: 0
    
    
    Done in 11.58s.

**Comment by llamatester** [comment]
The ipor adapter exports: 
        
 
 _______________________________________
    Dexs for 27/2/2024
    _______________________________________
    
    Version -> DERIVATIVES
    ---------
    ETHEREUM 👇
    Backfill start time: 16/8/2022
    NO METHODOLOGY SPECIFIED
    Timestamp: 1709078399 (2024-02-27T23:59:59.000Z)
    Daily volume: 0
    
    
    ARBITRUM 👇
    Backfill start time: 21/2/2024
    NO METHODOLOGY SPECIFIED
    Timestamp: 1709078399 (2024-02-27T23:59:59.000Z)
    Daily volume: 0
    
    
    Done in 9.79s.

**Comment by llamatester** [comment]
The ipor adapter exports: 
        
 
 _______________________________________
    Dexs for 27/2/2024
    _______________________________________
    
    Version -> DERIVATIVES
    ---------
    ETHEREUM 👇
    Backfill start time: 16/8/2022
    NO METHODOLOGY SPECIFIED
    Timestamp: 1709078399 (2024-02-27T23:59:59.000Z)
    Daily volume: 0
    
    
    ARBITRUM 👇
    Backfill start time: 21/2/2024
    NO METHODOLOGY SPECIFIED
    Timestamp: 1709078399 (2024-02-27T23:59:59.000Z)
    Daily volume: 671.51 k
    
    
    Done in 9.10s.

**Pull request merged**